src/commonmark/en/content/user/using-the-capture-app.md

```diff
@@ -429,48 +429,90 @@ The Capture app doesn't support tracker programs yet, but the tracker programs a
 
    ![](resources/images/capture_app/search-by-attributes-find-button.png)
 
-These steps will take you to the search page. There, given the configuration your organisation has in the maintenance app you will see the different attributes you can search with. An example of how this looks like is the following.
+    These steps will take you to the search page. There, based on the configuration of your organisation, will see the different attributes you can search with. An example of how this looks is the following.
 
-![](resources/images/capture_app/search-by-attributes-on-scope-program-overview-0.png)
+    ![](resources/images/capture_app/search-by-attributes-on-scope-program-overview-0.png)
 
-To execute a search now, 
+    To execute a search now: 
 
 1. Fill in the attributes you want to search with.
 
-2. Clcik the Search by attributes button.
+2. Click the **Search by attributes** button.
 
    ![](resources/images/capture_app/search-by-attributes-on-scope-program-overview-1.png)
+   
+3. The results of the search will be displayed as follows.
+
+   ![](resources/images/capture_app/search-by-attributes-on-scope-program-overview-2.png)
+    
+   In this list you can see the entries that match your search. For each entry you can have a total of three options. 
+    
+   a. You can choose to view the users dashboard by clicking the "View dashboard" button
+   
+   ![](resources/images/capture_app/search-by-attributes-on-scope-program-overview-5.png)
+    
+   b. You can view the the active enrollment of a user by clicking the "View active enrollment" button
+   
+   ![](resources/images/capture_app/search-by-attributes-on-scope-program-overview-3.png)
+    
+   c. You can re-enroll a user to the current program you are searching within.
+   
+   ![](resources/images/capture_app/search-by-attributes-on-scope-program-overview-4.png)
 
+    
 #### Fallback search
 
-Execute a full search as described above. Now if the search you have made has results you will be seeing your results.
+Execute a full search as described above. If the search you have made has results they will be displayed. However, the actual user you are searching for may be within a different program. In that case, you may want to extend the search to other programs. This is known as a fallback search.
 
-To execute a fallback search, press the button on the bottom saying "Search in all programs". 
-**Note: this is only possible in the scope of a Program.**
+To execute a fallback search, simply press the button on the bottom saying "Search in all programs". 
+
+> **Note**
```

Comment:
Format a note like this to make sure it is rendered nicely in our documentation.